### PR TITLE
Pipeline Moved to new Certification

### PR DESCRIPTION
**Pull request by ugras-ergun-sonarsource**
There are too many commits so I will squash them

### CODE REVIEW

File: pipeline/azure-pipelines.yml [hunk 1]
```diff
@@ -154,12 +170,25 @@ jobs:
     inputs:
       solution: $(BuildParameters.solution)
       vsVersion: 17.0
-      msbuildArgs: /p:VsTargetVersion=$(vsTargetVersion) /p:DeployExtension=false /p:SignArtifacts=$(SHOULD_SIGN) /p:AssemblyOriginatorKeyFile="$(snk.secureFilePath)" /p:pfxCertificatePath="$(pfx.secureFilePath)" /p:pfxPassword="$(pfxPassword2021-2023)" /p:pfxSha1=$(pfxSha1-2021-2023) /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId)  $(AdditionalMSBuildArgs)
+      msbuildArgs: /p:VsTargetVersion=$(vsTargetVersion) /p:DeployExtension=false /p:SignArtifacts=false /p:AssemblyOriginatorKeyFile="$(snk.secureFilePath)" /p:Sha1=$(Build.SourceVersion) /p:BuildNumber=$(Build.BuildId)  $(AdditionalMSBuildArgs)
```
Comment (ugras-ergun-sonarsource) [inline pipeline/azure-pipelines.yml:173]: Now instead of signing the artifact on build we sign them after the build.

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Some questions to look at

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM
What's the `-Override` result? Did you try another branch?

**Comment by ugras-ergun-sonarsource** [comment]
> LGTM What's the `-Override` result? Did you try another branch?

I ran on the branch with force sign parameter

File: pipeline/azure-pipelines.yml [hunk 2]
```diff
@@ -104,11 +99,32 @@ jobs:
...
+    name: SM_CLIENT_CERT
+    displayName: Download p12 file
     inputs:
-      secureFile: SonarSource-2021-2023.pfx
+      secureFile: digicert_authentication_certificate.p12
       retryCount: 5
+  
+  - task: SSMClientToolsSetup@1
+    displayName: Client Tools Setup
+    condition: eq(variables['SHOULD_SIGN'], 'true')
```
Comment (costin-zaharia-sonarsource) [inline pipeline/azure-pipelines.yml:110]: I'm not sure I understand who sets `SHOULD_SIGN` to true and when this happens. Could give me a hint? :)

Comment (pavel-mikula-sonarsource) [reply pipeline/azure-pipelines.yml:110]: https://github.com/SonarSource/sonarlint-visualstudio/blob/master/pipeline/scripts/should-sign-vsix.ps1#L43

Comment (costin-zaharia-sonarsource) [reply pipeline/azure-pipelines.yml:110]: Ok, now I got it. Thanks!

**Pull request merged**